src/reducers/index.js

```diff
@@ -12,9 +12,9 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {count: ++state.count}
```

Comment:
This _works_, however, you should careful with the way that you are handling this! `++` can mutate! 

A safer way would be:
`return {count: state.count + 1}`

Also, it isn't really necessary for this project, but to _REALLY_ make sure that you aren't mutating, you could even do:
`return {...state, count: state.count + 1}`